js/privatebin.js

```diff
@@ -1195,17 +1195,21 @@ jQuery.PrivateBin = (function($, sjcl, Base64, RawDeflate) {
          */
         function sendToShortener()
         {
-            var url = $shortenButton.data('shortener')+encodeURIComponent($pasteUrl.attr('href'));
+            var url = $shortenButton.data('shortener') + encodeURIComponent($pasteUrl.attr('href'));
 
             if($shortenButton.data('ajax'))
+            {
                 $.get(url, function(d) {
                     $pasteUrl.attr(d);
-                    $pasteUrl.html(d);
+                    $pasteUrl.html(d.text());
```

Comment:
Youz can use `text` directly on the element, instead of converting it three times here… :smile: